test-suite/micronaut-serialization/src/test/java/io/micronaut/configuration/metrics/serialization/MetricsMicronautSerializationTest.java

```diff
@@ -0,0 +1,27 @@
+package io.micronaut.configuration.metrics.serialization;
+
+import java.util.List;
+import java.util.Map;
+
+import io.micronaut.http.client.HttpClient;
+import io.micronaut.http.client.annotation.Client;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+@MicronautTest
+class MetricsMicronautSerializationTest {
```

Comment:
I assume you moved this to junit to be able to test it with native as well?

Reply:
correct